MAUI/ImageEditor/accessibility.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
```

Comment:
add accessibility and keyboard under single md file

https://help.syncfusion.com/xamarin/scheduler/accessibility

Reply:
Added